src/ts/constants/networks.ts

```diff
@@ -20,22 +20,22 @@ export const Alexander: Network = {
   genesisHash: '0xdcd1346701ca8396496e52aa2785b1748deb6db09551b72159dcb3e08991025b'
 }
 
-export const CharredCherry: Network = {
-  chain: Substrate,
-  name: 'Charred-Cherry',
-  rpcServer: 'wss://substrate-rpc.parity.io/',
-  txExplorer: 'https://polkadot.js.org/apps/#/explorer',
-  genesisHash: '0xxxxxxx' // TODO either fix me or remove CharredCherry
+export const TestNode: Network = {
```

Comment:
EdgewareTestNode?

Reply:
TestNode is certainly not a good name, but I think it'll be temporary. Hopefully Edgeware team will have a better name for their network later. Since its value provides some context and close to its declaration, hopefully it won't be too confusing.